packages/assert/package.json

```diff
@@ -9,6 +9,7 @@
   "scripts": {
     "build": "exit 0",
     "test": "ava",
+    "test-nyc": "nyc ava",
```

Comment:
Please prefer `"test:nyc"` for things that are modifiers of the base `"test"` script.